### DISCUSSION TITLE
Activating Wind on openfast

### DISCUSSION THREAD

**Discussion by FR4T3R** [OpenFAST/openfast#1473] [Q&A]
I am trying to run a simulation that only involves the wind on a spar type floating wind turbine 

i have previously ran simulations with regular and irregular waves changing this line of code 

![image](https://user-images.githubusercontent.com/126684439/222156564-be40f9f3-2a7e-43fd-a1b5-e0015c368ae7.png)

i assumed to activate the wind it was using this line of code 

![image](https://user-images.githubusercontent.com/126684439/222156888-4eba4d05-e6dc-471c-b587-887598af7a8f.png)

however when i ran openfast i got this error 

![image](https://user-images.githubusercontent.com/126684439/222157050-c76ea0a5-59c2-4202-af31-4895fa3bf68c.png)

can anyone help me understand where i am going wrong ? 

thank you !!

**Comment by jjonkman** [comment]
Dear @FR4T3R,

Similar questions have been asked and answered on this github site and on our forum before, e.g., here: https://github.com/OpenFAST/openfast/discussions/1281 and https://forums.nrel.gov/t/error-ff-wind-array-was-exhausted/1267.

Best regards,
